BSB_LAN/BSB_LAN.ino

```diff
@@ -1996,10 +1992,10 @@ void SerialPrintData(byte* msg) {
  *   Serial  instance
  * *************************************************************** */
 void SerialPrintRAW(byte* msg, byte len) {
-  for (int i=0;i<len;i++) {
-    SerialPrintHex(msg[i]);
-    printFmtToDebug(PSTR(" "));
-  }
+  int outBufLen = strlen(outBuf);
+  bin2hex(outBuf + outBufLen, msg, len, ' ');
+  printToDebug(outBuf + outBufLen);
+  outBuf[outBufLen] = 0;
```

Comment:
This code is less readable than the previous version. Does it have any significant advantage?